app/views/static_pages/dashboard_steps/community_rules.md

```diff
@@ -64,25 +23,13 @@ We encourage questions, so do not worry about asking a "stupid question". Howeve
 
 </div>
 
-#### 10. Be professional and have good etiquette
+#### Be professional and have good etiquette
 
 We expect everyone to handle themselves professionally and treat others respectfully, no matter how old you are or how much experience you have. To ensure that everyone has the same expectations, below are some specific guidelines:
 
 <div class="lesson-content__panel" markdown="1">
 
--   **Do not post personal information.** This applies to all sensitive information, as well as **your age** (unless you are an adult).
-
--   **No sexual or offensive usernames, nicknames, or profile pictures.** This detracts from a professional, friendly, safe and welcoming environment. Depending on the context, this may result in either a moderator changing your nickname or requesting you to change your profile picture, or an immediate ban.
-
--   **Do not compare time with others.** When people try to compare how much time it takes even to do TOP or even just one project, it tends to lead to feelings of inadequacy or a false sense of accomplishment. Everyone has different life circumstances and project completion expectations, so we discourage comparing time and instead focus on learning the material well.
-
--   **Only ping another user when necessary.** It is often not necessary to ping a specific person because everyone shares the responsibility to help others. If you choose to ping another user about a recent conversation, please wait patiently for a reply.
-
--   **Do not bomb the chat by sending multiple messages in a row.** Please type your entire comment or question in one message. To create a new line, use `shift + enter/return` on your computer or the `enter/return` button on your mobile device.
-
--   **Provide context when sharing a resource.** When posting a link to an article or youtube video, include **why** you are sharing it. Otherwise it will be marked as spam and deleted.
-
--   **Post your finished projects in #creations-showcase.** This channel is a great collection of finished projects. Feedback happens in the #creations-discussions channel. Everyone is welcome to provide feedback on a project that you have completed.
+-   **Do not post personal information.** This applies to all sensitive information, as well as **your age**.
```

Comment:
Is there a particular reason `(unless you are an adult)` part has been removed?

Reply:
Yes, because there has been discussion on how really no one should be discussing their age, so I thought this was a good time to remove this additional info.